src/content/app/species-selector/components/find-gene-instructions/FindGeneInstructions.scss

```diff
@@ -0,0 +1,47 @@
+@import 'src/styles/common';
```

Comment:
Something wrong with the styles resulting in an extra width:

https://user-images.githubusercontent.com/6834224/200786934-30ae0e20-067a-4dc3-bbbd-4185fd527f16.mov

Reply:
That should be fixed now ✅

Reply:
Is this really how this component should behave at a 980px width?

https://user-images.githubusercontent.com/6834224/201080787-8aff1836-907c-4339-9436-d1ea4151ee24.mov